CONTRIBUTING.md

```diff
@@ -1,72 +1,46 @@
 
-Thanks!
 
 **Contents**
 
+* [Reporting a security vulnerabilitiy](#security-vulnerability)
 * [Reporting general bugs](#reporting-general-bugs)
 * [Contributing code](#contributing-code)
 * [Contributing documentation and translations](#contributing-documentation-and-translations)
 
-## Reporting general bugs
-
-**If you are reporting a security vulnerability**, please consult the
-instructions in our SECURITY.md file before proceeding. If you have discovered
-an issue in AtoM that is **not** related to a security vulnerability, we
-welcome your reports.
-
-You can report general bugs in two ways: 
-
-* By creating a new post in the [AtoM user forum](https://groups.google.com/forum/#!forum/ica-atom-users)
-* By [opening an issue](https://github.com/artefactual/atom/issues) in our
-  GitHub repository
-
-Please be sure to include all the information we will need to reproduce the
-issue locally. This includes:
+## Security vulnerability
 
-* The version of AtoM you are using
-* Basic information about your installation environment, including PHP, MySQL,
-  Elasticsearch, and operating system versions
-* Steps to reproduce the issue
-* The resulting error or vulnerability and the expected outcome
-* If there are any error logs related to the issue, please include the
-  relevant parts as well
+**If you are reporting a security vulnerability**, refer yourself to the
+[SECURITY.md](./SECURITY.md) document. 
 
-You can find useful tips on how to find this information in the Troubleshooting 
-page in our documentation:
+## Reporting general bugs
 
-* https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/
+If you have discovered an issue in AtoM, we welcome your reports.
 
-Note as well that, as a community-driven open source project, we depend on our
-community to be able to maintain and develop AtoM. We are committed to
-including as many bug fixes as we can in each new release, but a confirmed
-report is not enough alone to guarantee that a fix will be included in the
-next release. If your institution is interested in sponsoring a fix, feel free
-to contact Artefactual Systems for an estimate - all sponsored development
-will be included in the next public release. For more information on how we
-develop and maintain AtoM, please see:
+Start by doing a search on [AtoM user forum](https://groups.google.com/forum/#!forum/ica-atom-users)
+and in the [AtoM GitHub issues](https://github.com/artefactual/atom/issues)
+to see if your question has been considered in the past.
 
-* https://wiki.accesstomemory.org/Development/Philosophy
+If nothing comes up, start a conversation by opening a new thread on the user
+forum. When your bug will have been better understood, you may be invited to
+open an issue on GitHub.
 
-We also welcome pull requests to fix issues!
+The [Troubleshooting guide](https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/) contains the information needed to open an issue.
```

Comment:
I don't think this is a good way to characterize the Troubleshooting page - if anything, I would mention it earlier as a possible way to RESOLVE their issue, prior to posting in the forum or opening an issue ticket. 

However, if you want to link it here as something that helps tell people _how_ to file an issue, then

1. link to the "getting support" section directly: https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/#getting-support
2. Consider doing a separate PR to update that section of the Troubleshooting docs, since it doesn't link to the AtoM issues at all (and perhaps also in light of my other more general comments here about what we should or shouldn't actually promise)
3. I would also suggest making the `open an issue` part of this sentence a hyperlink to the place to file issues in GitHub

Reply:
Maybe this:

```
The [Troubleshooting guide](https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/)
contains useful tips on how to find information needed to open an issue,
such as the [version of AtoM](https://www.accesstomemory.org/fr/docs/2.8/user-manual/administer/settings/#application-version),
accessing [errors logs and debug mode](https://www.accesstomemory.org/fr/docs/2.8/admin-manual/maintenance/troubleshooting/#troubleshooting-logs-debug),
and other information needed to [get support](https://www.accesstomemory.org/fr/docs/2.8/admin-manual/maintenance/troubleshooting/#getting-support).
```

Reply:
looks good, just make sure that those later links take out the `/fr/` and use the `/latest/` trick instead of specifying 2.8!